src/toil/test/src/promisedRequirementTest.py

```diff
@@ -123,6 +123,11 @@ def testPromisesWithJobStoreFileObjects(self):
             Job.Runner.startToil(F1, self.getOptions(self._createTempDir('testFiles')))
 
 
+        def testPromiseRequirementRace(self):
```

Comment:
What's the point? The test should be committed first, then the fix in a subsequent commit.